Dockerfile

```diff
@@ -15,4 +15,7 @@ RUN pip install -r requirements.txt
 
 ADD . /usr/src/app
 
+#Start flask dev server on port 5000
```

Comment:
I personally wouldn't leave any comments in code just sake of good "Clean Code" principles. Good code is like a good joke, it needs no explanation! (: